pages/u/[username]/card.tsx

```diff
@@ -55,6 +58,17 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     };
   }
 
+  const { data: userData, error } = await fetchApiData({
+    path: `users/${username}`,
+    pathValidator: () => safeParse(GitHubUserNameSchema, username).success,
+  });
+
+  if (error || !userData) {
```

Comment:
If it's an error that is not a 404, we should log the error via Sentry and let the 500 page load up.